linode/manager
# M3 4405 e2e support

How to verify that that the test works:

yarn up
yarn cy:debug
Choose open-support-ticket.spec.ts in the cypress test runner
Test should pass
Extra check (runs headless):

yarn cy:e2e -s ./cypress/integration/help & support/open-support-ticket.spec.ts
Result should be "✔ All specs passed!"
Make sure your .env contains:
REACT_APP_LOGIN_ROOT='https://login.linode.com'
REACT_APP_API_ROOT='https://api.linode.com/v4'
REACT_APP_APP_ROOT='http://localhost:3000'
REACT_APP_LAUNCH_DARKLY_ID='*************************' (CMR flag)
REACT_APP_CLIENT_ID='' (https://cloud.linode.com/profile/clients, copy ID)
MANAGER_OAUTH='******************************************************'
(create in https://cloud.linode.com/profile/tokens > "Add a Person Access Token", Save the token that appears in the modal, you will only see it once)
